pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools>=61.0", "build", "wheel"]
+
+[project]
+name = "spec_parser"
+version = "0.1.0"
+dependencies = [
```

Comment:
Is there value in leaving them out?

If you're wanting to avoid even hassling with installation when you just want to check, you can still run the main.py file; it's just done differently now.  I can add docs to the README to mention that if we want.